test/unit/test/pretty_printing_test.cpp

```diff
@@ -44,18 +44,27 @@ TEST(pretty_printing, gtest_output)
 
     EXPECT_EQ(gtest_str(std::make_tuple<int, int>(42, -10)), "(42, -10)"s);
     EXPECT_EQ(debug_str(std::make_tuple<int, int>(42, -10)), "(42,-10)"s);
-}
-
-TEST(pretty_printing, std_output)
-{
-    EXPECT_EQ(gtest_str(std::vector<std::vector<int>>{{0,1}, {2,3}, {1,2}, {0}}), "[[0,1],[2,3],[1,2],[0]]"s);
-    EXPECT_EQ(debug_str(std::vector<std::vector<int>>{{0,1}, {2,3}, {1,2}, {0}}), "[[0,1],[2,3],[1,2],[0]]"s);
 
+#if GTEST_INTERNAL_HAS_VARIANT // googletest >1.10.0
```

Comment:
I am not sure of googletest version semantics.
`GTEST_INTERNAL_HAS_VARIANT` is in there `master` branch, but I can't find it in there `v1.10.x` branch.

If I had to guess, I would say it only appears in `googletest >= 1.11.0`
and it will not be in any `googletest < 1.11.0` version. So if for what ever reason there is a 1.10.1 version I would assume they will not include it?